src/llama-kv-cache-unified.cpp

```diff
@@ -961,6 +963,7 @@ void llama_kv_cache_unified::set_input_kq_mask(ggml_tensor * dst, const llama_ub
     //      xxxxx-----
     //      xxxxx-----
     // To visualize the mask, see https://github.com/ggml-org/llama.cpp/pull/12615
+    // TODO: optimize this section
```

Comment:
Note for later: there is opportunity for optimizing the KQ mask initialization here. For large `n_kv` this section becomes measurable.

Reply:
This will be fixed in https://github.com/ggml-org/llama.cpp/pull/14600 after merging #14363